src/fields/field_fe.cc

```diff
@@ -259,6 +259,7 @@ void FieldFE<spacedim, Value>::cache_update(FieldValueCache<typename Value::elem
 {
     ASSERT( !boundary_dofs_ ).error("boundary field NOT supported!!\n");
     Armor::ArmaMat<typename Value::element_type, Value::NRows_, Value::NCols_> mat_value;
```

Comment:
TODO:
1. measure in proper context, proof if it is worth of 
2. more general implementation for other types of Mappings